client/src/pages/Location.js

```diff
@@ -77,16 +79,24 @@ const Location = () => {
               {locationByName.pic.map((locationPic) => (
                 <>
                   <Image
-                    key={locationPic.name}
-                    locationName={name}
+                    key={locationPic.link}
+                    locationname={name}
```

Comment:
not sure here, but wouldn't you also want locationPic.name for locationname?
Also use camelCase for locationName ;)

Reply:
No, sorry locationName and locationPic.name are two different things. the first one is the location name and the second one is the name of a picture. Going to change the camelcase in locationName 👍